flask_monitoringdashboard/static/pages/telemetry.html

```diff
@@ -0,0 +1,51 @@
+<div id="telemetry-consent" class="alert alert-info" ng-show="telemetryShow">
+    <strong>Telemetry Consent:</strong>
+    <p>Help us enhance our services by allowing anonymous telemetry collection. This data helps us understand usage
+        patterns and improve your experience. You have full control and can modify your choice at any time in the
```

Comment:
... anonymous usage collection...

... helps us understand which features are useful, and which can be retired - we are a small research project, and we can not afford to maintain features that are not useful... 

we could even make the reader realize that them helping with anonymous usage info is an effortless way to contribute to the project by adding something like this maybe?  

> We are also an open source project so if you can't help us with usage information, you can maybe help us with adopting and fixing some <a href="...">issues</a>?

Reply:
Very good insight, we'll discuss and improve this.